pom.xml

```diff
@@ -7,10 +7,10 @@
         <relativePath />
     </parent>
 
-    <url>https://github.com/jenkinsci/codesonar-plugin</url>
```

Comment:
URL is incorrect, it should not say eficode.  Looks like merge conflicts were not resolved correctly